client/engine/store/durablestore.go

```diff
@@ -266,6 +266,29 @@ func (ds *DurableStore) getChannelById(id types.Destination) (channel.Channel, e
 	return ch, nil
 }
 
+// GetChannelsByAppDefinition returns any channels that include the given participant
+func (ds *DurableStore) GetChannelsByAppDefinition(appDef types.Address) []*channel.Channel {
+	toReturn := []*channel.Channel{}
+	err := ds.channels.View(func(tx *buntdb.Tx) error {
+		err := tx.Ascend("", func(key, chJSON string) bool {
+			var ch channel.Channel
+			err := json.Unmarshal([]byte(chJSON), &ch)
+			if err != nil {
+				ds.checkError(err)
+			}
+
+			if ch.AppDefinition == appDef {
+				toReturn = append(toReturn, &ch)
+			}
+
+			return true // channel not found: continue looking
```

Comment:
This comment seems wrong: the channel may have been found. We do continue looking, of course.

Reply:
Resolved by https://github.com/statechannels/go-nitro/pull/1268/commits/79d587861afd733bc95cbdb634e6c635e7ee5521